device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml

```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: HPE
+model: ProLiant DX385 Gen10 Plus V2
+slug: hpe-proliant-dx385-gen10-plus-v2
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+front_image: true
+rear_image: true
+console-ports:
+  - name: VGA
+    type: other
+  - name: USB1
+    type: usb-a
+  - name: USB2
+    type: usb-a
```

Comment:
Are these used for console access? If not, these do not belong here.

Reply:
fixed